interfaces/portal/src/app/pages/snake/game/snake.component.ts

```diff
@@ -137,6 +147,11 @@ export class SnakeComponent implements AfterViewInit {
     ) {
       this.score.set(this.snakeBody.length - 3);
       this.isGameOver.set(true);
+      this.trackingService.trackEvent({
+        category: TrackingCategory.hiddenFeatures,
+        action: TrackingAction.showSnakeGameOver,
+        value: this.score(),
+      });
```

Comment:
Is this tracked per user? @elwinschmitz

Reply:
Not “specific logged-in user” really. 

You can see what other things that user has done on that day. 

And there’s a backlog-item for the “tracking per role/user” already somewhere.